### ISSUE TITLE
Accessibility: Buttons in share screen not labeled correct when using voice over

### ISSUE THREAD

**Issue by Falcopus1986** [signalapp/Signal-iOS#4801]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.
When you send a link to someone using the share button in another app the labels of some buttons are not correct when using voice-over.

### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug
- Open safari (or another app) and go to a website
- Click the share button
- Choose signal
- Choose one or more contacts to share with
- Than you have a "next" button but this is labeled as "Arrow right 24"
- In the next screen the "send" button is also labeled as "send solid 24"
- In this screen the "cancel" button is also labeled as "compose cancel"

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
The name of the buttons is not clear.....I also use the dutch version of signal and the labels are the same (so no translation)

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
I give the wrong and the corect labels
- "Arrow right 24" = "Next"
- "Send solid 24" = "send"
- "compose cancel" = "cancel"
### Device info

**Device**: iPhone XS

**iOS version**: 14.3

**Signal version:** 5.2.1.4
